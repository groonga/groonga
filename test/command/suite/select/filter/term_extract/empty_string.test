table_create Words TABLE_PAT_KEY ShortText --normalizer NormalizerAuto
load --table Words
[
{"_key": "groonga"}
]
select Words --filter '_key *T ""'
