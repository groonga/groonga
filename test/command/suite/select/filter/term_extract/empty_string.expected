table_create Words TABLE_PAT_KEY ShortText --normalizer NormalizerAuto
[[0,0.0,0.0],true]
load --table Words
[
{"_key": "groonga"}
]
[[0,0.0,0.0],1]
select Words --filter '_key *T ""'
[[0,0.0,0.0],[[[0],[["_id","UInt32"],["_key","ShortText"]]]]]
